modules/nf-core/thermorawfileparser/main.nf

```diff
@@ -0,0 +1,33 @@
+process THERMORAWFILEPARSER {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "bioconda::thermorawfileparser=1.4.2"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/thermorawfileparser%3A1.4.2--ha8f3691_0' :
+        'quay.io/biocontainers/thermorawfileparser:1.4.2--ha8f3691_0'    }"
+
+    input:
+    tuple val(meta), path(raw)
+
+    output:
+    tuple val(meta), path("*.mzML"), emit: mzML
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+    """
+    thermorawfileparser \\
+        -i=$raw \\
+        -f=2 \\
```

Comment:
It's a parameter for indexing the output file. It does not have to be able every time. Do you want me to remove it? In fact it shoul be a input parameter to be modified by the module user...But for simplicity it can be skipped for the moment. What do you think?